src/sys/ptrace/linux.rs

```diff
@@ -342,6 +347,9 @@ pub fn getregs(pid: Pid) -> Result<user_regs_struct> {
 }
 
 /// Get a particular set of user registers, as with `ptrace(PTRACE_GETREGSET, ...)`
+///
+/// Currently, in x86_64 platform, if the tracer is 64bit and tracee is 32bit, this function
+/// will return an error.
```

Comment:
Nit: we should document that it will return an EIO or users may think this errno comes from the underlying ptrace syscall.

Reply:
Updated